src/lib/layouts/workflow-run-layout.svelte

```diff
@@ -7,7 +7,8 @@
   import Header from '$lib/layouts/workflow-header.svelte';
   import Loading from '$holocene/loading.svelte';
   import { onDestroy, onMount } from 'svelte';
-  import { eventFilterSort, EventSortOrder } from '$lib/stores/event-view';
+  import { eventFilterSort } from '$lib/stores/event-view';
+  import type { EventSortOrder } from '$lib/stores/event-view';
```

Comment:
This can still be done in a single line I believe

`import { type EventSortOrder, eventFilterSort } from '$lib/stores/event-view';`